drivers/adcs/AdcsInterface.cpp

```diff
@@ -0,0 +1,59 @@
+#include "AdcsInterface.hpp"
+
+#include <linux/i2c-dev.h>
+#include <i2c/smbus.h>
+#include <sys/ioctl.h>
+#include <fcntl.h>
+#include <stddef.h>
+
+// TODO: Determine i2c bus number and write it here
+const char *i2c_dev_filename = "/dev/i2c-xx";
+// TODO: Find adcs i2c address
+const int adcs_address = 0;
+int i2c_file;
+
+struct Coordinates {
+    float latitude;
+    float longitude;
+};
+
+void AdcsInterface::EnableAdcs() {
```

Comment:
for functions that aren't constructors/destructors, please name them beginning with a lowercase.

eg. thisFunction()